### PR TITLE
NAPALM integration with SR OS 

### PR DESCRIPTION
**Pull request by ashah26**
This is an initial commit of integration NAPALM with SR OS with documentation

### CODE REVIEW

File: Install.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        netconf {
+            base-op-authorization {
+                kill-session true
+                lock true
+            }
+        }
+       ``` 
+2) Install napalm, by using command `pip install napalm`  
+
+3) From [NOKIA NAPALM REPO](https://github.com/nokia/napalm-SROS-driver/tree/develop) clone the github repository onto your local computer
```
Comment (hellt) [inline Install.md:56]: wrong url, it should be `https://github.com/nokia/napalm-SROS-driver`. Also there is no need to clone, as I mentioned before, the easy way to install from the git repo is to do `pip install https://github.com/hellt/napalm-sros-ssh/archive/master.zip`

File: License_sros_napalm_driver.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
```
Comment (hellt) [inline License_sros_napalm_driver.md:1]: I am not sure its a good idea to use a custom name for a license, why not use the standard LICENSE.md file?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+NAPALM (Network Automation and Programmability Abstraction Layer with Multivendor support) is a Python library that implements a set of functions to interact with different router vendor devices using a unified API.
+
+NAPALM supports several methods to connect to the devices, to manipulate configurations or to retrieve data.
+
+#### **SR OS**
+NAPALM integration is validated with a minimum of Nokia Service Router Operating System (SR OS) version 19.10R5. Releases beyond this have not been validated and should be by users before using the driver in labs and production on devices using different SR OS versions. Please contact the Nokia owners of this repository for additional information with respect to additional release validation.
+
+
+#### **Documentation**
+1) Please read the installation instruction in [INSTALL](https://github.com/nokia/napalm-SROS-driver/blob/develop/Install.md)
```
Comment (hellt) [inline README.md:13]: wrong url refers to the develop branch

File: README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+2) The main files included for Nokia SR OS driver are:
+     3) napalm_sros/sros.py: Overridden NAPALM methods to get the expected output from SR OS
+     4) napalm_sros/nc_filters.py: Filters defined to get data from SR OS using a NETCONF connection
+5) Mapping of various parameters of NAPALM output to Nokia SR OS can be found in this [DOCUMENT](https://github.com/nokia/napalm-SROS-driver/blob/develop/Summary_of_Methods.pdf)
```
Comment (hellt) [inline README.md:17]: wrong url refers to the develop branch

File: example_script.py [hunk 5]
```diff
@@ -0,0 +1,9 @@
+from napalm import get_network_driver
+
+driver = get_network_driver("sros")
+optional_args = {'port': 830}
+device = driver("138.120.181.55", "netconf", "n37c0nf20", 60, optional_args)
```
Comment (hellt) [inline example_script.py:5]: lets not have public IPs in the demo scripts

File: test/unit/sros/mock_data/get_arp_table.txt [hunk 6]
```diff
@@ -0,0 +1,40 @@
+<data xmlns="urn:ietf:params:xml:ns:netconf:base:1.0" xmlns:nc="urn:ietf:params:xml:ns:netconf:base:1.0">
+    <state xmlns="urn:nokia.com:sros:ns:yang:sr:state">
+        <router>
+            <router-name>management</router-name>
+            <interface>
+                <interface-name>management</interface-name>
+                <ipv4>
+                    <neighbor-discovery>
+                        <neighbor>
+                            <ipv4-address>138.120.180.1</ipv4-address>
```
Comment (hellt) [inline test/unit/sros/mock_data/get_arp_table.txt:10]: I would vet the test files to exclude public IPs and substitute them with private IPs

(search for `138.` pattern)

File: example_script.py (hunk 5 above)
Comment (ashah26) [reply example_script.py:5]: I replaced it mock ip address - 127.0.0.1

File: test/unit/sros/mock_data/get_arp_table.txt (hunk 6 above)
Comment (ashah26) [reply test/unit/sros/mock_data/get_arp_table.txt:10]: I replaces it to 10.10.10.1 private IP address

**Pull request merged**